src/language/__tests__/schema-kitchen-sink.graphql

```diff
@@ -41,6 +41,8 @@ union AnnotatedUnionTwo @onUnion = | A | B
 
 scalar CustomScalar
 
+scalar StringEncodedCustomScalar = String
```

Comment:
Yeah, I agree. `implements` seems like the closest thing, `:` seems reasonable as well